### PR TITLE
Validation: Added Missing Descriptions

### PR DESCRIPTION
**Pull request by dereklh7**
Added missing metadata descriptions.

### CODE REVIEW

**Comment by dereklh7** [comment]
@stevemunk @JuanPapas @rbrundritt 

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

**Comment by dereklh7** [comment]
@stevemunk @JuanPapas @rbrundritt

Checking in on the status of this PR to see if there are any changes that need to be implemented for this PR to be approved and merged. Thanks!

**Comment by stevemunk** [comment]
@dereklh7 Please undo changes/remove these files from this PR. They were all removed in PR230:

BingMaps/sdk-native/map-services-api/maproutefinderresult-class.md
BingMaps/sdk-native/map-services-api/maprouteleg-class.md
BingMaps/sdk-native/map-services-api/maproutemaneuver-class.md
BingMaps/sdk-native/map-services-api/maproutemaneuverkind-enumeration.md
BingMaps/sdk-native/map-services-api/maprouteoptimization-enumeration.md
BingMaps/sdk-native/map-services-api/maprouterestrictions-enumeration.md
BingMaps/sdk-native/map-services-api/routewaypoint-class.md
BingMaps/sdk-native/map-services-api/routewaypointtype-enumeration.md
BingMaps/sdk-native/map-services-api/trafficcongestion-enumeration.md

**Comment by dereklh7** [comment]
@stevemunk I've reverted fixes made to the files you called out. Please let me know if there are other issues that need to be addressed for this PR.

**Comment by stevemunk** [comment]
@dereklh7 Something is wrong... These files are now appearing as new files being added in the PR. I think you need to sync with main, which should remove those files from your branch, otherwise your PR will try to add them back.

**Comment by dereklh7** [comment]
@stevemunk Apologies for the error on my end on the previous commit. I have removed the files in my local repo, which should fix all issues in this PR. Please let me know if there are any remaining issues preventing approval and merge for this PR. Thanks!

File: BingMaps/spatial-data-services/data-source-management-api/load-data-source-dataflow/index.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Load Data Source Dataflow | Microsoft Docs"
+description: "Describes the Load Data Source Dataflow API and provides a table that gives a description for various resources."
```
Comment (stevemunk) [inline BingMaps/spatial-data-services/data-source-management-api/load-data-source-dataflow/index.md:3]:
```suggestion
description: The overview page for the Load Data Source Dataflow section contains a description of the Load Data Source Dataflow API, used to create a data source that contains entity data for a user-specified entity type, and provides links to and descriptions of each article in this section.

```

File: BingMaps/spatial-data-services/data-source-management-api/load-data-source-dataflow/load-data-source-dataflow-sample-code-csharp.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Load Data Source Dataflow Sample Code (C#) | Microsoft Docs"
+description: "Provides Load Data Source Dataflow sample C# code for classes that upload spatial data and check status of dataflow jobs and support classes."
```
Comment (stevemunk) [inline BingMaps/spatial-data-services/data-source-management-api/load-data-source-dataflow/load-data-source-dataflow-sample-code-csharp.md:3]:
```suggestion
description: C# sample code demonstrating how to upload entity data to a data source using the Load Data Source Dataflow
```

**Review by stevemunk — CHANGES_REQUESTED** [review]

**Comment by dereklh7** [comment]
@stevemunk I've implemented requested fixes. Let me know if there are any remaining issues preventing approval and merge for this PR. Thanks!

**Pull request merged**